config/locales/en.yml

```diff
@@ -31,7 +31,7 @@ en:
     weve_processed_your_spreadsheet: We’ve processed your spreadsheet
     guide_to_collecting_mobile_information: Guide to collecting mobile information
     guide_for_distributing_bt_vouchers: A guide for distributing vouchers to families
-    start: Increase children’s internet access
+    start: Get help with technology
```

Comment:
I think the `<title>` page for this will end up being:
`Get help with technology - Get help with technology`